### DISCUSSION TITLE
Version 9 questions

### DISCUSSION THREAD

**Discussion by bruce-c-liu** [jeffreylanters/react-unity-webgl#458] [Question and Answer]
Hi! v8 of the library has been working great for us, but we were thinking of upgrading to v9.

1. Could you please elaborate on "Major improvements to the performance of the module" written in the release notes?

2. Are there any breaking changes? What's the upgrade path?

**Comment by jeffreylanters** [comment]
Hi. The most notable changes for the user is that the unity context is now a hook, while under the hood the entire codebase is rewritten. It's also worth nothing that *most* new features will only roll out under the major version 9 releases.

I hope this helps you decide wether it is worth updating.
